blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DBasicNumericIO.java

```diff
@@ -0,0 +1,33 @@
+package com.rackspacecloud.blueflood.io.datastax;
```

Comment:
OH! It's fine then, just add the comment to make it more clear the difference between DBasicNumericIO and DSimpleNumberIO.